var/spack/repos/builtin/packages/nest/package.py

```diff
@@ -13,7 +13,7 @@ class Nest(CMakePackage):
     than on the exact morphology of individual neurons."""
 
     homepage = "http://www.nest-simulator.org"
-    url      = "https://github.com/nest/nest-simulator/releases/download/v2.12.0/nest-2.12.0.tar.gz"
+    url      = "https://github.com/nest/nest-simulator/archive/v2.12.0.tar.gz"
```

Comment:
This new URL will result in different checksums for all versions. Can you update those checksums?